spec.md

```diff
@@ -443,7 +443,7 @@ It is therefore RECOMMENDED that implementations avoid such strings.
 
 | Response Field | Type | Description |
 | --- | --- | --- |
-| name* | string | A CLI-friendly name of the service. MUST only contain alphanumeric characters, periods, and hyphens (no spaces). MUST be unique across all service objects returned in this response. MUST be a non-empty string. |
+| name* | string | The name of the service. MUST be unique across all service objects returned in this response. MUST be a non-empty string. Using a CLI-friendly name is RECOMMENDED. |
```

Comment:
Can we define here what a CLI-friendly name is? 
For example something like: "Using a CLI-friendly name, which only contains alphanumeric characters, periods, hyphens and no spaces is RECOMMENDED."

Reply:
We define it here: https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#body
But it might not stand-out enough. Should we consider adding it to our "terminology" section?  Then we can just add an href to that section from here.

Reply:
@fmui is this still an issue? a blocker?